batch_job/src/work_queue_factory.c

```diff
@@ -602,16 +602,16 @@ struct jx *factory_to_jx(struct list *masters, struct list *foremen, int submitt
 	jx_insert_integer(j, "workers_to_connect", to_connect);
 
 	struct jx *ms = jx_array(NULL);
-	if(masters && list_size(masters) > 0)
+	if(managers && list_size(managers) > 0)
 	{
 		struct jx *m;
-		list_first_item(masters);
-		while((m = list_next_item(masters)))
+		list_first_item(managers);
+		while((m = list_next_item(managers)))
 		{
-			jx_array_append(ms, master_to_jx(m));
+			jx_array_append(ms, manager_to_jx(m));
 		}
 	}
-	jx_insert(j, jx_string("masters"), ms);
+	jx_insert(j, jx_string("managers"), ms);
```

Comment:
Will this break catalog lookups?

Reply:
I can add both. However, that "masters" is not used anywhere else.